app/controllers/SubmissionsController.scala

```diff
@@ -51,8 +52,15 @@ class SubmissionsController @Inject()(
   def amend(id: String): Action[AnyContent] = authenticate.async { implicit request =>
     customsDeclareExportsConnector.findDeclaration(id) flatMap {
       case Some(declaration) =>
+        val currentDateTime = Instant.now()
+        val newDeclaration = declaration.copy(
```

Comment:
can we move this logic to companion object of declaration ?

Reply:
done